README.md

```diff
@@ -313,6 +313,7 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 * [TorchServe](https://github.com/pytorch/serve) - A flexible and easy to use tool for serving PyTorch models.
 * [Triton Inference Server](https://github.com/triton-inference-server/server) - Provides an optimized cloud and edge inferencing solution.
 * [Vespa](https://github.com/vespa-engine/vespa) - Store, search, organize and make machine-learned inferences over big data at serving time.
+* [Geniusrise](https://docs.geniusrise.ai) AI microservices framework & ecosystem. Host inference APIs, bulk inference and fine tune text, vision, audio and multi-modal models.
```

Comment:
Thanks @ixaxaar, could you please:
- Add it in alphabetical order;
- Add an hyphen before the description;
- Make the description shorter to avoid line breaking?

Reply:
-    Add it in alphabetical order;
    -  done
-    Add an hyphen before the description;
    -  done
-    Make the description shorter to avoid line breaking?
    -  done